src/worker.js

```diff
@@ -7,20 +7,17 @@ import BatchManager from './utils/BatchManager';
 
 let closing = false;
 
-export default (app, config, onServer, workerId) => {
-  let server;
-
-  // ===== Middleware =========================================================
+export const attachMiddleware = (app, config) => {
```

Comment:
I can rewrite it in CJS and it will be expected (and common) pattern/behavior :) Would it work for you? :))

I'd love to move it to the entry point, but it would require file shuffling and I wasn't sure how big of a change you're ok with. :)